<!DOCTYPE html><html lang="en-au"><head><meta charset="utf-8"><meta http-equiv="X-UA-Compatible" content="IE=edge"><meta name="viewport" content="width=device-width,initial-scale=1"><title>CV - Dinnage Lab</title><meta name="description" content="Curriculum Vitae Last updated: March 17, 2025 Research Assistant Professor Department of Biological Sciences Florida International University 11200 SW 8th Street, Miami, Florida, United States&hellip;"><meta name="generator" content="Publii Open-Source CMS for Static Site"><link rel="canonical" href="https://rdinnager.github.io/dinnage_lab_website/cv.html"><link rel="alternate" type="application/atom+xml" href="https://rdinnager.github.io/dinnage_lab_website/feed.xml" title="Dinnage Lab - RSS"><link rel="alternate" type="application/json" href="https://rdinnager.github.io/dinnage_lab_website/feed.json" title="Dinnage Lab - JSON"><meta property="og:title" content="CV"><meta property="og:site_name" content="Dinnage Lab"><meta property="og:description" content="Curriculum Vitae Last updated: March 17, 2025 Research Assistant Professor Department of Biological Sciences Florida International University 11200 SW 8th Street, Miami, Florida, United States&hellip;"><meta property="og:url" content="https://rdinnager.github.io/dinnage_lab_website/cv.html"><meta property="og:type" content="article"><link rel="preload" href="https://rdinnager.github.io/dinnage_lab_website/assets/dynamic/fonts/jetbrainsmono/jetbrainsmono.woff2" as="font" type="font/woff2" crossorigin><link rel="preload" href="https://rdinnager.github.io/dinnage_lab_website/assets/dynamic/fonts/jetbrainsmono/jetbrainsmono-italic.woff2" as="font" type="font/woff2" crossorigin><link rel="stylesheet" href="https://rdinnager.github.io/dinnage_lab_website/assets/css/style.css?v=9ac3045b1a727c7938c187aa0847fb44"><script type="application/ld+json">{"@context":"http://schema.org","@type":"Article","mainEntityOfPage":{"@type":"WebPage","@id":"https://rdinnager.github.io/dinnage_lab_website/cv.html"},"headline":"CV","datePublished":"2025-03-19T15:49-04:00","dateModified":"2025-03-19T15:49-04:00","description":"Curriculum Vitae Last updated: March 17, 2025 Research Assistant Professor Department of Biological Sciences Florida International University 11200 SW 8th Street, Miami, Florida, United States&hellip;","author":{"@type":"Person","name":"Russell Dinnage","url":"https://rdinnager.github.io/dinnage_lab_website/authors/russell-dinnage/"},"publisher":{"@type":"Organization","name":"Russell Dinnage"}}</script><noscript><style>img[loading] {
                    opacity: 1;
                }</style></noscript></head><body class="page-template"><div class="container"><header class="header"><div class="header__logo"><a class="logo" href="https://rdinnager.github.io/dinnage_lab_website/">Dinnage Lab</a></div><nav class="navbar js-navbar"><button class="navbar__toggle js-toggle" aria-label="Menu" aria-haspopup="true" aria-expanded="false"><span class="navbar__toggle-box"><span class="navbar__toggle-inner">Menu</span></span></button><ul class="navbar__menu"><li><a href="https://rdinnager.github.io/dinnage_lab_website/research-program.html" target="_self">My Research</a></li><li class="has-submenu"><a href="https://rdinnager.github.io/dinnage_lab_website/software.html" target="_self" aria-haspopup="true">R Software</a><ul class="navbar__submenu level-2" aria-hidden="true"><li><a href="https://rdinnager.github.io/phyf/" target="_self">phyf</a></li><li><a href="https://rdinnager.github.io/fibre/" target="_self">fibre</a></li><li><a href="https://rdinnager.github.io/slimr/" target="_self">slimr</a></li><li><span class="is-separator">----- Co-developed -----</span></li><li><a href="https://daijiang.github.io/phyr/" target="_self">phyr</a></li><li><a href="https://github.com/danlwarren/ENMTools" target="_self">ENMTools</a></li></ul></li><li><a href="https://rdinnager.github.io/dinnage_lab_website/publications.html" target="_self">Publications</a></li><li class="active"><a href="https://rdinnager.github.io/dinnage_lab_website/cv.html" target="_self">CV</a></li><li><a href="https://rdinnager.github.io/dinnage_lab_website/people.html" target="_self">People</a></li></ul></nav></header><main class="content page"><article class="post"><header><h1 class="post__title">CV</h1></header><div class="post__entry"><h1>Curriculum Vitae</h1><p><em>Last updated: March 17, 2025</em></p><h2>Russell Dinnage, Ph.D.</h2><p>Research Assistant Professor<br>Department of Biological Sciences<br>Florida International University<br>11200 SW 8th Street, Miami, Florida, United States<br>Phone: +1 786 956 9089<br>Email: rdinnage@fiu.edu<br>Website: <a href="https://rdinnager.github.io/dinnage_lab_website/">rdinnager.github.io/dinnage_lab_website/</a><br>Twitter: <a href="https://twitter.com/ecologician">@ecologician</a><br>GitHub: <a href="https://github.com/rdinnager">github.com/rdinnager</a><br>Google Scholar: <a href="https://scholar.google.com/citations?user=GU6p2KoAAAAJ">scholar.google.com/citations?user=GU6p2KoAAAAJ</a></p><h2>Academic Positions</h2><p><strong>Research Assistant Professor</strong><br>Florida International University<br>08/2021-present</p><p><strong>Postdoctoral Researcher</strong><br>University of Canberra<br>12/2019-06/2021<br>Project: Understanding the effects of rapid environmental change on the adaptive potential of desert animals using landscape genomics simulations and deep learning<br>Supervisors: Stephen Sarre, Bernd Gruber, and Richard Duncan</p><p><strong>Postdoctoral Researcher</strong><br>Australian National University<br>08/2015-12/2019<br>Project: Understanding the influence of macroevolution on local scale ecological processes by testing the predictions of the niche filling model<br>Supervisors: Marcel Cardillo and Lindell Bromham</p><p><strong>Postdoctoral Researcher</strong><br>CSIRO Ecosystem Sciences<br>2012-2015<br>Project: Ecological metagenomics of Kimberley rainforest patch invertebrates<br>Supervisors: Owain Edwards and Raphael Didham</p><h2>Education</h2><p><strong>Ph.D. in Ecology and Evolutionary Biology</strong><br>University of Toronto<br>2007-2013<br>Thesis Title: The effects of phylogenetic diversity in plant communities on herbivores and herbivory<br>Supervisor: Peter Abrams</p><p><strong>B.S. in Ecology (major) and Zoology (minor)</strong><br>University of Toronto<br>2002-2006</p><h2>Current Grants</h2><p><strong>National Science Foundation - Division of Environmental Biology: $198,141</strong><br>2024 - 2026<br>PIs: Russell Dinnage<br>Title: Species Distribution Modeling on the A.I. frontier: Deep generative models for powerful, general and accessible SDM<br>Link: <a href="https://www.nsf.gov/awardsearch/showAward?AWD_ID=2329701">nsf.gov/awardsearch/showAward?AWD_ID=2329701</a></p><h2>Publications</h2><h3>Preprints</h3><ul><li>Dinnage, R. (2024). NicheFlow: Towards a foundation model for Species Distribution Modelling. bioRxiv, 2024-10. doi: <a href="https://doi.org/10.1101/2024.10.15.618541">https://doi.org/10.1101/2024.10.15.618541</a><br>Status: Revising for special issue of Methods in Ecology and Evolution entitled: "Conservation, Ecology and Artificial Intelligence: Advances and Symbiotic Solutions"</li><li>Dinnage, R., Grady, E., Neal, N., Deck, J., Denny, E., Walls, R., Seltzer, C., Guralnick, R. and Li, D. (2024). PhenoVision: A framework for automating and delivering research-ready plant phenology data from field images. bioRxiv, 2024-10. doi: <a href="https://doi.org/10.1101/2024.10.10.617505">https://doi.org/10.1101/2024.10.10.617505</a><br>Status: Under review at Methods in Ecology and Evolution</li><li>Dinnage, R. (2023). How many variables does Wordclim have, really? Generative AI unravels the intrinsic dimension of bioclimatic variables. bioRxiv, doi: <a href="https://doi.org/10.1101/2023.06.12.544623">https://doi.org/10.1101/2023.06.12.544623</a><br>Status: Revising for Ecology.</li></ul><h3>Journal Publications</h3><ol><li>Dinnage, R. and Kleineberg, M. (2025). Generative AI helps extract ecological meaning from the complex three dimensional shapes of bird bills. PLoS Computational Biology. In Press.</li><li>Simonsen, A., and Dinnage, R. (2025). Rhizobia mutualists contribute to phylogenetic clustering and legume community assembly globally. Journal of Ecology. 00:1–14.</li><li>Galán-Acedo, C., Verde Arregoitia, L.D., Arasa-Gisbert, R., Auliz-Ortiz, D., Saldivar-Burrola, L.L., Gouveia, S.F., Correia, I., Rosete-Vergés, F.A., Dinnage, R. and Villalobos, F. (2024). Global primary predictors of extinction risk in primates. Proceedings of the Royal Society B. 291, no. 2032: 20241905.</li><li>Dinnage, R., Sarre, S.D., Duncan, R.P., Dickman, C.R., Edwards, S.V., Greenville, A.C., Wardle, G.M. and Gruber, B. (2024). slimr: An R package for tailor‐made integrations of data in population genomic simulations over space and time. Molecular Ecology Resources, 24(3), e13916.</li><li>Hill, P., Dickman, C.R., Dinnage, R., Duncan, R.P., Edwards, S.V., Greenville, A., Sarre, S.D., Stringer, E.J., Wardle, G.M., &amp; Gruber, B. (2023). Episodic population fragmentation and gene flow reveal a trade-off between heterozygosity and allelic richness. Molecular Ecology, 32(24), 6766-6776.</li><li>Skirgård, H., Haynie, H.J., Blasi, D.E., Hammarström, H., Collins, J., Latarche, J.J., Lesage, J., Weber, T., Witzlack-Makarevich, A., Passmore, S. and Chira, A., Maurits, L., Dinnage, R., et al. (2023). Grambank reveals the importance of genealogical constraints on linguistic diversity and highlights the impact of language loss. Science Advances, 9(16), eadg6175.</li><li>Cardillo, M., Skeels, A., &amp; Dinnage, R. (2023). Priorities for conserving the world's terrestrial mammals based on over-the-horizon extinction risk. Current Biology, 33(7), 1381-1388.</li><li>Jiang, G. F., Li, S. Y., Dinnage, R., Cao, K. F., Simonin, K. A., &amp; Roddy, A. B. (2023). Diverse mangroves deviate from other angiosperms in their genome size, leaf cell size, and cell packing density relationships. Annals of Botany, 131(2), 347-360.</li><li>Machado, F. F., Jardim, L., Dinnage, R., Brito, D., &amp; Cardillo, M. (2023). Diet disparity and diversity predict extinction risk in primates. Animal Conservation, 26(3), 331-339.</li><li>Bromham L, Dinnage R, Skirgård H, Ritchie A, Cardillo M, Meakins F, Greenhill S, Hua X. (2022) Global predictors of language endangerment and the future of linguistic diversity. Nature ecology &amp; evolution. 6(2):163-73.</li><li>Skeels, A., Dinnage, R., Medina, I., &amp; Cardillo, M. (2021). Ecological interactions shape the evolution of flower color in communities across a temperate biodiversity hotspot. Evolution Letters, 5(3), 277-289.</li><li>Bromham, L., Skeels, A., Schneemann, H., Dinnage, R., &amp; Hua, X. (2021). There is little evidence that spicy food in hot countries is an adaptation to reducing infection risk. Nature Human Behaviour, 5, 878–891.</li><li>Warren, D.L., Matzke, N.J., Cardillo, M., Baumgartner, J., Beaumont, L.J., Turelli, M., Glor, R., Huron, N.A., Simões, M., Iglesias, T.L., Dinnage, R. (2021). ENMTools 1.0: an R package for comparative ecological biogeography. Ecography, 44(4), 504-511.</li><li>Ritchie, A. M., Hua, X., Cardillo, M., Yaxley, K. J., Dinnage, R., &amp; Bromham, L. (2021). Phylogenetic diversity metrics from molecular phylogenies: modelling expected degree of error under realistic rate variation. Diversity and Distributions, 27(1), 164-178.</li><li>Li, D., Dinnage, R., Nell, L.R., Helmus, M.R., Ives, A.R. (2020) phyr: An r package for phylogenetic species‐distribution modelling in ecological communities. Methods in Ecology and Evolution, 11: 1455– 1463.</li><li>Dinnage, R., Skeels, A., and Cardillo, M. (2020). Spatiophylogenetic modelling of extinction risk reveals evolutionary distinctiveness and brief flowering period as risk factors in a diverse hotspot plant genus. Proceedings of the Royal Society B. 287, 20192817.</li><li>Dinnage R., Simonsen A.K., Barrett L.G., Cardillo M., Raisbeck-Brown N., Thrall P.H., and Prober, S.M., 2019. Larger plants promote a greater diversity of symbiotic nitrogen-fixing soil bacteria associated with an Australian endemic legume. Journal of Ecology, 107(2), 977-991. * These authors contributed equally to this manuscript.</li><li>Cardillo M., Dinnage R., McAlister W., 2019. The relationship between environmental niche breadth and geographic range size across plant species. Journal of Biogeography, 46:97–109.</li><li>Warren, D.L., Beaumont, L.J., Dinnage, R. and Baumgartner, J.B., 2018. New methods for measuring ENM breadth and overlap in environmental space. Ecography, 41: 1–3.</li><li>Cadotte, M.W., Livingstone, S.W., Yasui, S.L.E., Dinnage, R., Li, J.T., Marushia, R., Santangelo, J. and Shu, W., 2017. Explaining ecosystem multifunction with evolutionary models. Ecology, 98(12), pp.3175-3187.</li><li>Simonsen, A.K., Dinnage, R., Barrett, L.G., Prober, S.M. and Thrall, P.H., 2017. Symbiosis limits establishment of legumes outside their native range at a global scale. Nature Communications, 8. * These authors contributed equally to this manuscript.</li><li>Bromham, L., Dinnage, R., Hua, X. (2016). Interdisciplinary research has consistently lower funding success. Nature, 534(7609): 684-687. doi: 10.1038/nature18315.</li><li>Dinnage, R. (2013). Phylogenetic diversity of plants alters the effect of species richness on invertebrate herbivory. PeerJ, 1, e93. doi:10.7717/peerj.93</li><li>Cadotte, M.W., Dinnage, R., and Tilman, D. (2012). Phylogenetic diversity promotes ecosystem stability. Ecology. 93(Supplement): S223–S233. doi:10.1890/11-0426.1</li><li>Dinnage, R., Cadotte, M.W., Haddad, N.M., Crutsinger, G.M., and Tilman, D. (2012). Diversity of plant evolutionary lineages promotes arthropod diversity. Ecology Letters. 15(11): 1308-1317.</li><li>Dinnage R. (2009). Disturbance alters the phylogenetic composition and structure of plant communities in an old field system. PLoS ONE 4(9): e7071.</li><li>Abrams, P.A., Rueffler, C., and Dinnage, R. (2008). Competition-similarity relationships and the nonlinearity of competitive effects in consumer-resource systems. American Naturalist, 172(4): 463-474.</li><li>Johnson, M.T.J., Dinnage, R., Zhou, A.Y., and Hunter, M.D. (2008). Environmental variation has stronger effects than plant genotype on competition among plant species. Journal of Ecology, 96: 947–955.</li></ol><p><strong>Statistics:</strong> Total Citation Count: 2893; H-index: 19; i10-index: 22; Mean Citations per Article: ~103</p><h2>Academic Service</h2><ul><li>Associate Editor - Methods in Ecology and Evolution (IF = 6.3), January 2025-present</li></ul><h2>Selected Presentations</h2><ul><li>Dinnage, R. (November 2024). BioSeminar: Modeling the Evolution of Integrated Complex Phenotypes across Lineages: Organisms and Languages Alike. Florida International University, Biology Departmental Seminar, Miami, Florida, USA.</li><li>Dinnage, R. (September 2023). Invited Talk: Surfing the grammar manifold: Ancestral state estimation and evolutionary rate estimation of Grambank characters on a phylogeny using latent generative models. Max Planck Institute for Evolutionary Anthropology, Leipzig, Germany. Recording available at <a href="https://www.youtube.com/watch?v=p3JbiSR8dNs">youtube.com/watch?v=p3JbiSR8dNs</a></li><li>Dinnage, R. (November 2022). Invited Talk: The Shape of Things: Using deep learning to model shape data. Harvard University, Cambridge, MA, USA.</li><li>Dinnage, R. (November 2020). Accepted Talk: Autodecoding Evolution: Exploring phylogenetic deep learning for ancestral reconstruction of traits with arbitrarily high dimensionality and complexity. Phylomania 2020. Online (hosted in Hobart, Tasmania, Australia)</li></ul><h2>Teaching</h2><table border="1" cellspacing="0" cellpadding="5"><tbody><tr><th>Course</th><th>Semester</th></tr><tr><td>Species Distribution Modeling: Understanding the past and future of biodiversity with machine learning - Florida International University</td><td>Fall 2022, Fall 2023</td></tr><tr><td>Evolution - Florida International University</td><td>Fall 2024</td></tr></tbody></table><p><strong>Species Distribution Modeling:</strong> A combined undergraduate (PCB4932) and graduate (BSC6936) course on species distribution modeling, which also includes an advanced introduction to machine learning in general.</p><p><strong>Evolution:</strong> An undergraduate core course (PCB 4674), providing a comprehensive introduction to the field of evolutionary biology, with a focus on understanding evolution as a process that creates phenotypic change within lineages of organisms. Using a process-based perspective, the course explores the fundamental mechanisms that drive evolutionary change, including natural selection, genetic drift, mutation, and gene flow.</p><h2>Previous Awards</h2><h3>Grants</h3><p><strong>Centre for Biodiversity Analysis (CBA) Ignition Grant: $10,000</strong><br>2016 - 2018<br>PIs: Marcel Cardillo, Gavin Huttley, Owain Edwards<br>Title: Characterizing the evolutionary and ecological diversity of invertebrates in the Monsoonal vine thickets of the Kimberley</p><p><strong>Centre for Biodiversity Analysis (CBA) Ignition Grant: $10,000</strong><br>2015 - 2017<br>PIs: Anna Simonsen, Marcel Cardillo, Luke Barrett, Suzanne Prober, Peter Thrall<br>Title: Environmental Drivers of Acacia-Associated Symbiotic Microbe Diversity</p><h3>Graduate</h3><p><strong>NSERC Alexander Graham Bell Canadian Graduate Scholarship – Doctoral (CGS-D): $35,000</strong><br>Sept 2010 – Sept 2012</p><p><strong>Ontario Graduate Scholarship (OGS): $15,000</strong><br>Sept 2009 – Sept 2010</p><p><strong>Canadian Society for Ecology and Evolution Annual Meeting Best Poster - 2nd Prize: $300</strong><br>May 2009</p><h2>Academic References</h2><ul><li>Scott V. Edwards, Professor and Curator, Harvard University, sedwards@fas.harvard.edu</li><li>Robert Guralnick, Professor and Curator, University of Florida, robgur@gmail.com</li><li>Stephen Sarre, Professor, University of Canberra, Stephen.Sarre@canberra.edu.au</li><li>Marcel Cardillo, Professor, Australian National University, marcel.cardillo@anu.edu.au</li><li>Lindell Bromham, Professor, Australian National University, lindell.bromham@anu.edu.au</li><li>Marc Cadotte, Professor, University of Toronto: mcadotte@utsc.utoronto.ca</li><li>Peter Abrams, Professor Emeritus, University of Toronto: peter.abrams@utoronto.ca</li></ul></div></article></main><footer class="footer"><div class="footer__inner"><div class="footer__copyright"><p>© 2024 Powered by Publii CMS :: <a href="https://github.com/panr/hugo-theme-terminal" target="_blank" rel="noopener">Theme</a> ported by the <a href="https://getpublii.com/customization-service/" target="_blank" rel="noopener">Publii Team</a></p></div></div></footer></div><script defer="defer" src="https://rdinnager.github.io/dinnage_lab_website/assets/js/scripts.min.js?v=c2232aa7558e9517946129d2a1b8c770"></script><script>window.publiiThemeMenuConfig={mobileMenuMode:'sidebar',animationSpeed:300,submenuWidth: 'auto',doubleClickTime:500,mobileMenuExpandableSubmenus:true,relatedContainerForOverlayMenuSelector:'.top'};</script><script>var images = document.querySelectorAll('img[loading]');
        for (var i = 0; i < images.length; i++) {
            if (images[i].complete) {
                images[i].classList.add('is-loaded');
            } else {
                images[i].addEventListener('load', function () {
                    this.classList.add('is-loaded');
                }, false);
            }
        }</script></body></html>