<!DOCTYPE html><html lang="en-au"><head><meta charset="utf-8"><meta http-equiv="X-UA-Compatible" content="IE=edge"><meta name="viewport" content="width=device-width,initial-scale=1"><title>Publications - Dinnage Lab</title><meta name="description" content="Last updated: March 17, 2025 Our lab's research appears in diverse journals covering ecological, evolutionary, and computational topics. Below are our publications organized by research&hellip;"><meta name="generator" content="Publii Open-Source CMS for Static Site"><link rel="canonical" href="https://rdinnager.github.io/dinnage_lab_website/publications.html"><link rel="alternate" type="application/atom+xml" href="https://rdinnager.github.io/dinnage_lab_website/feed.xml" title="Dinnage Lab - RSS"><link rel="alternate" type="application/json" href="https://rdinnager.github.io/dinnage_lab_website/feed.json" title="Dinnage Lab - JSON"><meta property="og:title" content="Publications"><meta property="og:site_name" content="Dinnage Lab"><meta property="og:description" content="Last updated: March 17, 2025 Our lab's research appears in diverse journals covering ecological, evolutionary, and computational topics. Below are our publications organized by research&hellip;"><meta property="og:url" content="https://rdinnager.github.io/dinnage_lab_website/publications.html"><meta property="og:type" content="article"><link rel="preload" href="https://rdinnager.github.io/dinnage_lab_website/assets/dynamic/fonts/jetbrainsmono/jetbrainsmono.woff2" as="font" type="font/woff2" crossorigin><link rel="preload" href="https://rdinnager.github.io/dinnage_lab_website/assets/dynamic/fonts/jetbrainsmono/jetbrainsmono-italic.woff2" as="font" type="font/woff2" crossorigin><link rel="stylesheet" href="https://rdinnager.github.io/dinnage_lab_website/assets/css/style.css?v=9ac3045b1a727c7938c187aa0847fb44"><script type="application/ld+json">{"@context":"http://schema.org","@type":"Article","mainEntityOfPage":{"@type":"WebPage","@id":"https://rdinnager.github.io/dinnage_lab_website/publications.html"},"headline":"Publications","datePublished":"2025-03-19T15:41-04:00","dateModified":"2025-03-19T15:41-04:00","description":"Last updated: March 17, 2025 Our lab's research appears in diverse journals covering ecological, evolutionary, and computational topics. Below are our publications organized by research&hellip;","author":{"@type":"Person","name":"Russell Dinnage","url":"https://rdinnager.github.io/dinnage_lab_website/authors/russell-dinnage/"},"publisher":{"@type":"Organization","name":"Russell Dinnage"}}</script><noscript><style>img[loading] {
                    opacity: 1;
                }</style></noscript></head><body class="page-template"><div class="container"><header class="header"><div class="header__logo"><a class="logo" href="https://rdinnager.github.io/dinnage_lab_website/">Dinnage Lab</a></div><nav class="navbar js-navbar"><button class="navbar__toggle js-toggle" aria-label="Menu" aria-haspopup="true" aria-expanded="false"><span class="navbar__toggle-box"><span class="navbar__toggle-inner">Menu</span></span></button><ul class="navbar__menu"><li><a href="https://rdinnager.github.io/dinnage_lab_website/research-program.html" target="_self">My Research</a></li><li class="has-submenu"><a href="https://rdinnager.github.io/dinnage_lab_website/software.html" target="_self" aria-haspopup="true">R Software</a><ul class="navbar__submenu level-2" aria-hidden="true"><li><a href="https://rdinnager.github.io/phyf/" target="_self">phyf</a></li><li><a href="https://rdinnager.github.io/fibre/" target="_self">fibre</a></li><li><a href="https://rdinnager.github.io/slimr/" target="_self">slimr</a></li><li><span class="is-separator">----- Co-developed -----</span></li><li><a href="https://daijiang.github.io/phyr/" target="_self">phyr</a></li><li><a href="https://github.com/danlwarren/ENMTools" target="_self">ENMTools</a></li></ul></li><li class="active"><a href="https://rdinnager.github.io/dinnage_lab_website/publications.html" target="_self">Publications</a></li><li><a href="https://rdinnager.github.io/dinnage_lab_website/cv.html" target="_self">CV</a></li><li><a href="https://rdinnager.github.io/dinnage_lab_website/people.html" target="_self">People</a></li></ul></nav></header><main class="content page"><article class="post"><header><h1 class="post__title">Publications</h1></header><div class="post__entry"><p><em>Last updated: March 17, 2025</em></p><p>Our lab's research appears in diverse journals covering ecological, evolutionary, and computational topics. Below are our publications organized by research themes, followed by a chronological list of all publications.</p><p><a href="https://scholar.google.com/citations?user=GU6p2KoAAAAJ">View all publications on Google Scholar →</a></p><h2>Recent Publications</h2><h3>2025</h3><ul><li><strong>Dinnage, R.</strong> and Kleineberg, M. (2025). Generative AI helps extract ecological meaning from the complex three dimensional shapes of bird bills. <em>PLoS Computational Biology</em>. In Press.</li><li>Simonsen, A., and <strong>Dinnage, R.</strong> (2025). Rhizobia mutualists contribute to phylogenetic clustering and legume community assembly globally. <em>Journal of Ecology</em>. 00:1–14.</li></ul><h3>2024</h3><ul><li>Galán-Acedo, C., Verde Arregoitia, L.D., Arasa-Gisbert, R., Auliz-Ortiz, D., Saldivar-Burrola, L.L., Gouveia, S.F., Correia, I., Rosete-Vergés, F.A., <strong>Dinnage, R.</strong> and Villalobos, F. (2024). Global primary predictors of extinction risk in primates. <em>Proceedings of the Royal Society B</em>. 291, no. 2032: 20241905.</li><li><strong>Dinnage, R.</strong>, Sarre, S.D., Duncan, R.P., Dickman, C.R., Edwards, S.V., Greenville, A.C., Wardle, G.M. and Gruber, B. (2024). slimr: An R package for tailor‐made integrations of data in population genomic simulations over space and time. <em>Molecular Ecology Resources</em>, 24(3), e13916.</li></ul><h2>Publications by Research Theme</h2><h3>Foundation Models &amp; Biological Understanding</h3><ul><li><strong>Dinnage, R.</strong> (2024). NicheFlow: Towards a foundation model for Species Distribution Modelling. <em>bioRxiv</em>, 2024-10. doi: <a href="https://doi.org/10.1101/2024.10.15.618541">https://doi.org/10.1101/2024.10.15.618541</a> (Revising for special issue of Methods in Ecology and Evolution)</li><li><strong>Dinnage, R.</strong> (2023). How many variables does Wordclim have, really? Generative AI unravels the intrinsic dimension of bioclimatic variables. <em>bioRxiv</em>, doi: <a href="https://doi.org/10.1101/2023.06.12.544623">https://doi.org/10.1101/2023.06.12.544623</a>. (Revising for Ecology)</li><li><strong>Dinnage, R.</strong> and Kleineberg, M. (2025). Generative AI helps extract ecological meaning from the complex three dimensional shapes of bird bills. <em>PLoS Computational Biology</em>. In Press.</li></ul><h3>High-Throughput Phenomics &amp; Digital Specimens</h3><ul><li><strong>Dinnage, R.</strong>, Grady, E., Neal, N., Deck, J., Denny, E., Walls, R., Seltzer, C., Guralnick, R. and Li, D. (2024). PhenoVision: A framework for automating and delivering research-ready plant phenology data from field images. <em>bioRxiv</em>, 2024-10. doi: <a href="https://doi.org/10.1101/2024.10.10.617505">https://doi.org/10.1101/2024.10.10.617505</a> (Under review at Methods in Ecology and Evolution)</li></ul><h3>Phylogenetic Comparative Methods</h3><ul><li><strong>Dinnage, R.</strong>, Skeels, A., &amp; Cardillo, M. (2020). Spatiophylogenetic modelling of extinction risk reveals evolutionary distinctiveness and brief flowering period as threats in a hotspot plant genus. <em>Proceedings of the Royal Society B</em>, 287(1926), 20192817.</li><li>Cardillo, M., Skeels, A., &amp; <strong>Dinnage, R.</strong> (2023). Priorities for conserving the world's terrestrial mammals based on over-the-horizon extinction risk. <em>Current Biology</em>, 33(7), 1381-1388.</li><li>Machado, F. F., Jardim, L., <strong>Dinnage, R.</strong>, Brito, D., &amp; Cardillo, M. (2023). Diet disparity and diversity predict extinction risk in primates. <em>Animal Conservation</em>, 26(3), 331-339.</li></ul><h3>Phylogenetic Community Ecology</h3><ul><li>Cadotte, M.W., <strong>Dinnage, R.</strong> and Tilman, D. (2012). Phylogenetic diversity promotes ecosystem stability. <em>Ecology</em>, 93: S223-S233.</li><li><strong>Dinnage, R.</strong>, Cadotte, M. W., Haddad, N. M., Crutsinger, G. M., &amp; Tilman, D. (2012). Diversity of plant evolutionary lineages promotes arthropod diversity. <em>Ecology Letters</em>, 15(11), 1308-1317.</li><li><strong>Dinnage R.</strong> (2009). Disturbance alters the phylogenetic composition and structure of plant communities in an old field system. <em>PLoS ONE</em> 4(9): e7071.</li><li>Simonsen, A., and <strong>Dinnage, R.</strong> (2025). Rhizobia mutualists contribute to phylogenetic clustering and legume community assembly globally. <em>Journal of Ecology</em>. 00:1–14.</li></ul><h3>Species Distribution Modelling</h3><ul><li>Li, D, <strong>Dinnage, R</strong>, Nell, LR, Helmus, MR, Ives, AR. (2020). phyr: An r package for phylogenetic species-distribution modelling in ecological communities. <em>Methods in Ecology and Evolution</em>. 11: 1455–1463.</li><li>Warren, D.L., Matzke, N.J., Cardillo, M., Baumgartner, J.B., Beaumont, L.J., Turelli, M., Glor, R.E., Huron, N.A., Simões, M., Iglesias, T.L., Piquet, J.C. and <strong>Dinnage, R.</strong> (2021). ENMTools 1.0: an R package for comparative ecological biogeography. <em>Ecography</em>, 44: 504-511.</li><li>Cardillo, M, <strong>Dinnage, R.</strong>, McAlister, W. (2019). The relationship between environmental niche breadth and geographic range size across plant species. <em>Journal of Biogeography</em>. 46: 97–109.</li><li><strong>Dinnage, R.</strong>, Simonsen, A. K., Barrett, L. G., Cardillo, M., Raisbeck‐Brown, N., Thrall, P. H., &amp; Prober, S. M. (2019). Larger plants promote a greater diversity of symbiotic nitrogen‐fixing soil bacteria associated with an Australian endemic legume. <em>Journal of Ecology</em>, 107(2), 977-991.</li></ul><h3>Cultural Comparative Analysis</h3><ul><li>Bromham, L., <strong>Dinnage, R.</strong>, Skirgård, H. et al. (2022). Global predictors of language endangerment and the future of linguistic diversity. <em>Nature Ecology &amp; Evolution</em>. 6, 163–173.</li><li>Bromham, L., Skeels, A., Schneemann, H., <strong>Dinnage, R.</strong>, &amp; Hua, X. (2021). There is little evidence that spicy food in hot countries is an adaptation to reducing infection risk. <em>Nature Human Behaviour</em>, 5(7), 878-891.</li><li>Bromham, L., <strong>Dinnage, R.</strong> &amp; Hua, X. (2016). Interdisciplinary research has consistently lower funding success. <em>Nature</em> 534, 684–687.</li><li>Skirgård, H., Haynie, H.J., Blasi, D.E., Hammarström, H., Collins, J., Latarche, J.J., Lesage, J., Weber, T., Witzlack-Makarevich, A., Passmore, S., Chira, A., Maurits, L., <strong>Dinnage, R.</strong>, et al. (2023). Grambank reveals the importance of genealogical constraints on linguistic diversity and highlights the impact of language loss. <em>Science Advances</em>, 9(16), eadg6175.</li></ul><h3>Simulation and Theoretical EEB</h3><ul><li><strong>Dinnage, R.</strong>, Sarre, S.D., Duncan, R.P., Dickman, C.R., Edwards, S.V., Greenville, A.C., Wardle, G.M. and Gruber, B. (2024). slimr: An R package for tailor‐made integrations of data in population genomic simulations over space and time. <em>Molecular Ecology Resources</em>, 24(3), e13916.</li><li>Hill, P., Dickman, C.R., <strong>Dinnage, R.</strong>, Duncan, R.P., Edwards, S.V., Greenville, A., Sarre, S.D., Stringer, E.J., Wardle, G.M., &amp; Gruber, B. (2023). Episodic population fragmentation and gene flow reveal a trade-off between heterozygosity and allelic richness. <em>Molecular Ecology</em>, 32(24), 6766-6776.</li><li>Abrams, P. A., Rueffler, C., &amp; <strong>Dinnage, R.</strong> (2008). Competition-similarity relationships and the nonlinearity of competitive effects in consumer-resource systems. <em>The American Naturalist</em>, 172(4), 463-474.</li></ul><h2>Citations and Impact</h2><p>Total Citation Count: 2893<br>H-index: 19<br>i10-index: 22<br>Mean Citations per Article: ~103</p><h2>Preprints</h2><ul><li><strong>Dinnage, R.</strong> (2024). NicheFlow: Towards a foundation model for Species Distribution Modelling. <em>bioRxiv</em>, 2024-10. doi: <a href="https://doi.org/10.1101/2024.10.15.618541">https://doi.org/10.1101/2024.10.15.618541</a></li><li><strong>Dinnage, R.</strong>, Grady, E., Neal, N., Deck, J., Denny, E., Walls, R., Seltzer, C., Guralnick, R. and Li, D. (2024). PhenoVision: A framework for automating and delivering research-ready plant phenology data from field images. <em>bioRxiv</em>, 2024-10. doi: <a href="https://doi.org/10.1101/2024.10.10.617505">https://doi.org/10.1101/2024.10.10.617505</a></li><li><strong>Dinnage, R.</strong> (2023). How many variables does Wordclim have, really? Generative AI unravels the intrinsic dimension of bioclimatic variables. <em>bioRxiv</em>, doi: <a href="https://doi.org/10.1101/2023.06.12.544623">https://doi.org/10.1101/2023.06.12.544623</a></li></ul></div></article></main><footer class="footer"><div class="footer__inner"><div class="footer__copyright"><p>© 2024 Powered by Publii CMS :: <a href="https://github.com/panr/hugo-theme-terminal" target="_blank" rel="noopener">Theme</a> ported by the <a href="https://getpublii.com/customization-service/" target="_blank" rel="noopener">Publii Team</a></p></div></div></footer></div><script defer="defer" src="https://rdinnager.github.io/dinnage_lab_website/assets/js/scripts.min.js?v=c2232aa7558e9517946129d2a1b8c770"></script><script>window.publiiThemeMenuConfig={mobileMenuMode:'sidebar',animationSpeed:300,submenuWidth: 'auto',doubleClickTime:500,mobileMenuExpandableSubmenus:true,relatedContainerForOverlayMenuSelector:'.top'};</script><script>var images = document.querySelectorAll('img[loading]');
        for (var i = 0; i < images.length; i++) {
            if (images[i].complete) {
                images[i].classList.add('is-loaded');
            } else {
                images[i].addEventListener('load', function () {
                    this.classList.add('is-loaded');
                }, false);
            }
        }</script></body></html>