<!DOCTYPE html><html lang="en-au"><head><meta charset="utf-8"><meta http-equiv="X-UA-Compatible" content="IE=edge"><meta name="viewport" content="width=device-width,initial-scale=1"><title>My Research - Dinnage Lab</title><meta name="description" content="Broadly my lab is focused on the intersection between EEB (ecology and evolutionary biology) and computer science (in the form of Artificial Intelligence, e.g. A.I. research) A.I. → EBB - How deep learning techniques can be brought into EEB in a deeper way (no pun intended)&hellip;"><meta name="generator" content="Publii Open-Source CMS for Static Site"><link rel="canonical" href="https://rdinnager.github.io/dinnage_lab_website/research-themes.html"><link rel="alternate" type="application/atom+xml" href="https://rdinnager.github.io/dinnage_lab_website/feed.xml"><link rel="alternate" type="application/json" href="https://rdinnager.github.io/dinnage_lab_website/feed.json"><meta property="og:title" content="My Research"><meta property="og:site_name" content="Dinnage Lab"><meta property="og:description" content="Broadly my lab is focused on the intersection between EEB (ecology and evolutionary biology) and computer science (in the form of Artificial Intelligence, e.g. A.I. research) A.I. → EBB - How deep learning techniques can be brought into EEB in a deeper way (no pun intended)&hellip;"><meta property="og:url" content="https://rdinnager.github.io/dinnage_lab_website/research-themes.html"><meta property="og:type" content="article"><style>:root{--body-font:-apple-system,BlinkMacSystemFont,"Segoe UI",Roboto,Oxygen,Ubuntu,Cantarell,"Fira Sans","Droid Sans","Helvetica Neue",Arial,sans-serif,"Apple Color Emoji","Segoe UI Emoji","Segoe UI Symbol";--heading-font:-apple-system,BlinkMacSystemFont,"Segoe UI",Roboto,Oxygen,Ubuntu,Cantarell,"Fira Sans","Droid Sans","Helvetica Neue",Arial,sans-serif,"Apple Color Emoji","Segoe UI Emoji","Segoe UI Symbol";--logo-font:-apple-system,BlinkMacSystemFont,"Segoe UI",Roboto,Oxygen,Ubuntu,Cantarell,"Fira Sans","Droid Sans","Helvetica Neue",Arial,sans-serif,"Apple Color Emoji","Segoe UI Emoji","Segoe UI Symbol";--menu-font:-apple-system,BlinkMacSystemFont,"Segoe UI",Roboto,Oxygen,Ubuntu,Cantarell,"Fira Sans","Droid Sans","Helvetica Neue",Arial,sans-serif,"Apple Color Emoji","Segoe UI Emoji","Segoe UI Symbol"}</style><link rel="stylesheet" href="https://rdinnager.github.io/dinnage_lab_website/assets/css/style.css?v=895e9719ad61f7cc4c086e0852f6f803"><script type="application/ld+json">{"@context":"http://schema.org","@type":"Article","mainEntityOfPage":{"@type":"WebPage","@id":"https://rdinnager.github.io/dinnage_lab_website/research-themes.html"},"headline":"My Research","datePublished":"2021-07-29T22:21","dateModified":"2023-02-07T13:21","description":"Broadly my lab is focused on the intersection between EEB (ecology and evolutionary biology) and computer science (in the form of Artificial Intelligence, e.g. A.I. research) A.I. → EBB - How deep learning techniques can be brought into EEB in a deeper way (no pun intended)&hellip;","author":{"@type":"Person","name":"Russell Dinnage","url":"https://rdinnager.github.io/dinnage_lab_website/authors/russell-dinnage/"},"publisher":{"@type":"Organization","name":"Russell Dinnage"}}</script></head><body><div class="site-container"><header class="top" id="js-header"><a class="logo" href="https://rdinnager.github.io/dinnage_lab_website/">Dinnage Lab</a><nav class="navbar js-navbar"><button class="navbar__toggle js-toggle" aria-label="Menu" aria-haspopup="true" aria-expanded="false"><span class="navbar__toggle-box"><span class="navbar__toggle-inner">Menu</span></span></button><ul class="navbar__menu"><li class="active"><a href="https://rdinnager.github.io/dinnage_lab_website/research-themes.html" target="_self">My Research</a></li><li><a href="https://rdinnager.github.io/dinnage_lab_website/gallery.html" target="_self">Gallery</a></li><li class="has-submenu"><a href="https://rdinnager.github.io/dinnage_lab_website/r-software.html" target="_self" aria-haspopup="true">R Software</a><ul class="navbar__submenu level-2" aria-hidden="true"><li><a href="https://rdinnager.github.io/phyf/" target="_self">phyf</a></li><li><a href="https://rdinnager.github.io/fibre/" target="_self">fibre</a></li><li><a href="https://rdinnager.github.io/slimr/" target="_self">slimr</a></li><li><span class="is-separator">----- Co-developed -----</span></li><li><a href="https://daijiang.github.io/phyr/" target="_self">phyr</a></li><li><a href="https://github.com/danlwarren/ENMTools" target="_self">ENMTools</a></li></ul></li><li><a href="https://scholar.google.com/citations?user&#x3D;GU6p2KoAAAAJ" target="_self">Publications</a></li></ul></nav></header><main><article class="post"><div class="hero"><header class="hero__content"><div class="wrapper"><h1>My Research</h1></div></header></div><div class="wrapper post__entry"><p>Broadly my lab is focused on the intersection between EEB (ecology and evolutionary biology) and computer science (in the form of Artificial Intelligence, e.g. A.I. research)<br><br><strong>A.I. → EBB</strong> - How deep learning techniques can be brought into EEB in a deeper way (no pun intended) than simply as a (powerful) function approximator. Instead, can we use A.I. techniques as an aid in understanding complex high dimensional biological data, by the application of deep generative models.</p><div style="width: 100%; height: 0; padding-bottom: 50%; position: relative;"><div class="post__iframe"><iframe loading="lazy" width="100%" height="100%" style="position: absolute;" src="https://giphy.com/embed/yuOIGQmtwE6z9eahcg" frameborder="0" class="giphy-embed" allowfullscreen="allowfullscreen" data-mce-fragment="1"></iframe></div></div><p><a href="https://giphy.com/gifs/yuOIGQmtwE6z9eahcg">via GIPHY</a></p><figure class="post__image"><img loading="lazy" style="color: var(--text-primary-color); font-family: var(--editor-font-family); font-size: inherit; font-weight: var(--font-weight-normal); outline: 3px solid rgba(var(--color-primary-rgb), 0.55)  !important;" src="https://rdinnager.github.io/dinnage_lab_website/media/posts/1/Beetle_ganspace_demo_1_small.gif" alt="Animation of Beetle morphology in latent space" width="800" height="444"></figure><br><br><strong>EEB → A.I. - </strong>I'm also interested in the opposite flow of ideas, from EEB to computer science. In particular, developing ways to incorporate prior knowledge of biological structure (or inductive biases in machine learning lingo) into A.I. models (starting with phylogeny).<br><br><strong>EEB ↔ A.I. -</strong> Lastly I am currently interested in studying certain computational algorithms as examples of 'artificial' eco-evolutionary systems, and applying EEB research methods to them. I am hoping this will form an #ArtificialLife component to my research, a new research area for me. What is an eco-evolutionary system?<p></p><h2>Eco-evolutionary Systems</h2><p>From a biological perspective, my past and future work is focused on understanding eco-evolutionary systems, and I use computational tools that I develop to ask these questions (largely in R, but also Python or Julia). </p><p>An eco-evolutionary system is a system with contemporaneously interacting individuals which are connected invisibly and to varying degrees through a set of evolutionary processes that has occurred over possibly long periods of time. Much of my work also has a conservation or restoration focus. I am interested in how advanced computational and statistical methods can help preserve communities of irreplaceably organisms and the evolutionary history they represent.</p><p>Within this very broad umbrella, my past work falls into several sub-categories that I will list below, along with some representative publications (or preprints or software). There is usually lots of cross-flow between these categories, and so sometimes a publications placement is a somewhat arbitrary choice of which it is most closely aligned with.</p><h2>Themes and Some Publications</h2><p> </p><p><strong>1) Phylogenetic Comparative Methods -</strong> How species' traits are are determined by their evolutionary history (and contemporary interactions).</p><ul><li><strong>Dinnage, R.</strong>, Skeels, A., &amp; Cardillo, M. (2020). Spatiophylogenetic modelling of extinction risk reveals evolutionary distinctiveness and brief flowering period as threats in a hotspot plant genus. Proceedings of the Royal Society B, 287(1926), 20192817. <a href="https://doi.org/10.1098/rspb.2019.2817">https://doi.org/10.1098/rspb.2019.2817</a> </li><li>Upcoming paper: <strong>Dinnage, R.</strong> and Kleineberg, M. (2023) Deep Representation Learning Helps Extract Ecological and Evolutionary Meaning in the Complex Three Dimensional Shapes of Bird Bills<figure class="post__image"><img loading="lazy" src="https://rdinnager.github.io/dinnage_lab_website/media/posts/1/gif_1_shrunk.gif" alt="Evolution of beak shape across a phylogeny of birds" width="1024" height="728"></figure></li><li>Soon to be released R packages:<ul><li><a href="https://rdinnager.github.io/phyf/">phyf</a> and <a href="https://rdinnager.github.io/fibre/">fibre</a></li></ul></li></ul><p><strong>2) Phylogenetic Community Ecology</strong> - How long term evolutionary processes constrains and shapes the interactions of coexisting communities of species.</p><ul><li>Cadotte, M.W., <strong>Dinnage, R.</strong> and Tilman, D. (2012), Phylogenetic diversity promotes ecosystem stability. Ecology, 93: S223-S233.<br><a href="https://doi.org/10.1890/11-0426.1">https://doi.org/10.1890/11-0426.1</a></li><li><strong>Dinnage, R.</strong>, Cadotte, M. W., Haddad, N. M., Crutsinger, G. M., &amp; Tilman, D. (2012). Diversity of plant evolutionary lineages promotes arthropod diversity. Ecology letters, 15(11), 1308-1317. <a href="https://doi.org/10.1111/j.1461-0248.2012.01854.x">https://doi.org/10.1111/j.1461-0248.2012.01854.x</a></li><li><strong>Dinnage R.</strong> (2009) Disturbance alters the phylogenetic composition and structure of plant communities in an old field system. PLoS ONE 4(9): e7071. <a href="https://doi.org/10.1371/journal.pone.0007071">https://doi.org/10.1371/journal.pone.0007071</a></li></ul><p><strong>3) Species Distribution Modelling </strong>- How species' biology determines their interaction with the environment and each other, while taking into account their evolutionary connections (including spatial processes)</p><ul><li><strong>Dinnage, R.</strong>, Simonsen, A. K., Barrett, L. G., Cardillo, M., Raisbeck‐Brown, N., Thrall, P. H., &amp; Prober, S. M. (2019). Larger plants promote a greater diversity of symbiotic nitrogen‐fixing soil bacteria associated with an Australian endemic legume. Journal of Ecology, 107(2), 977-991. <a href="https://doi.org/10.1111/1365-2745.13083">https://doi.org/10.1111/1365-2745.13083</a> </li><li><strong>Dinnage, R.</strong>, &amp; Cardillo, M. (2018).* Habitat loss is information loss: Species distribution models are compromised in anthropogenic landscapes. bioRxiv, 258038. <a href="https://doi.org/10.1101/258038">https://doi.org/10.1101/258038</a> </li><li>Li, D, <strong>Dinnage, R</strong>, Nell, LR, Helmus, MR, Ives, AR. phyr: An r package for phylogenetic species-distribution modelling in ecological communities. Methods Ecol Evol. 2020; 11: 1455– 1463. <a href="https://doi.org/10.1111/2041-210X.13471">https://doi.org/10.1111/2041-210X.13471</a></li><li>Warren, D.L., Matzke, N.J., Cardillo, M., Baumgartner, J.B., Beaumont, L.J., Turelli, M., Glor, R.E., Huron, N.A., Simões, M., Iglesias, T.L., Piquet, J.C. and <strong>Dinnage, R.</strong> (2021), ENMTools 1.0: an R package for comparative ecological biogeography. Ecography, 44: 504-511. <a href="https://doi.org/10.1111/ecog.05485">https://doi.org/10.1111/ecog.05485</a></li><li>Cardillo, M, <strong>Dinnage, R.</strong>, McAlister, W. The relationship between environmental niche breadth and geographic range size across plant species. J Biogeogr. 2019; 46: 97– 109. <a href="https://doi.org/10.1111/jbi.13477">https://doi.org/10.1111/jbi.13477</a></li><li>Warren, D.L., Beaumont, L.J., <strong>Dinnage, R.</strong> and Baumgartner, J.B. (2019), New methods for measuring ENM breadth and overlap in environmental space. Ecography, 42: 444-446. <a href="https://doi.org/10.1111/ecog.03900">https://doi.org/10.1111/ecog.03900</a> </li></ul><p><br><strong>4) Cultural Comparative Analysis</strong> - How the evolutionary connections between human cultures can be used, estimated and/or accounted for when asking contemporary question about human behaviour and culture. This is largely driven by collaborations with linguists and cultural evolution specialists, since I am a biologist by training, but the methods we use have a lot of overlap!</p><ul><li>Bromham, L., <strong>Dinnage, R.</strong>, Skirgård, H. et al. Global predictors of language endangerment and the future of linguistic diversity. Nat Ecol Evol6, 163–173 (2022). <a href="https://doi.org/10.1038/s41559-021-01604-y">https://doi.org/10.1038/s41559-021-01604-y</a></li><li>Bromham, L., Skeels, A., Schneemann, H., <strong>Dinnage, R.</strong>, &amp; Hua, X. (2021). There is little evidence that spicy food in hot countries is an adaptation to reducing infection risk. Nature Human Behaviour, 5(7), 878-891. <a href="https://doi.org/10.1038/s41562-020-01039-8">https://doi.org/10.1038/s41562-020-01039-8</a> </li><li>Bromham, L., <strong>Dinnage, R.</strong> &amp; Hua, X. Interdisciplinary research has consistently lower funding success. Nature 534, 684–687 (2016). <a href="https://doi.org/10.1038/nature18315">https://doi.org/10.1038/nature18315</a></li><li>Skirgård, H., Haynie, H. J., Blasi, D. E., Hammarström, H., Collins, J., Latarche, J. J., <strong>et al.</strong> (2022). Grambank reveals the importance of genealogical constraints on linguistic diversity and highlights the impact of language loss. SocArXiv Papers, grh45. <a href="https://doi.org/10.31235/osf.io/grh45">https://doi.org/10.31235/osf.io/grh45</a> </li></ul><p><strong>5) Simulation and Theoretical EEB -</strong> Understanding the processes involved in eco-evolutionary systems and their long-term outcome and consequences.</p><ul><li><strong>Dinnage, R.</strong>, Sarre, S. D., Duncan, R. P., Dickman, C. R., Edwards, S. V., Greenville, A., ... &amp; Gruber, B. (2021). * slimr: An R package for integrating data and tailor-made population genomic simulations over space and time. bioRxiv, 2021-08. <a href="https://doi.org/10.1101/2021.08.05.455258">https://doi.org/10.1101/2021.08.05.455258</a> <ul><li>Associated R package: <a href="https://rdinnager.github.io/slimr/">slimr</a></li></ul></li><li>Abrams, P. A., Rueffler, C., &amp; <strong>Dinnage, R.</strong> (2008). Competition-similarity relationships and the nonlinearity of competitive effects in consumer-resource systems. The American Naturalist, 172(4), 463-474.<a href="https://doi.org/10.1086/590963"> https://doi.org/10.1086/590963</a> </li></ul><p>* Preprint</p><p> </p><p> </p><p> </p></div><footer class="wrapper post__footer"><div class="post__share"></div></footer></article><nav class="post__nav"><div class="post__nav-inner"><div class="post__nav-next"><a href="https://rdinnager.github.io/dinnage_lab_website/gallery.html" class="invert post__nav-link" rel="next"><span>Next</span> Gallery </a><svg width="1.041em" height="0.416em" aria-hidden="true"><use xlink:href="https://rdinnager.github.io/dinnage_lab_website/assets/svg/svg-map.svg#arrow-next"/></svg></div></div></nav></main><footer class="footer"><div class="footer__copyright"><p>Powered by <a href="https://getpublii.com" target="_blank" rel="nofollow noopener">Publii</a></p></div><button class="footer__bttop js-footer__bttop" aria-label="Back to top"><svg><title>Back to top</title><use xlink:href="https://rdinnager.github.io/dinnage_lab_website/assets/svg/svg-map.svg#toparrow"/></svg></button></footer></div><script>window.publiiThemeMenuConfig = {    
        mobileMenuMode: 'sidebar',
        animationSpeed: 300,
        submenuWidth: 'auto',
        doubleClickTime: 500,
        mobileMenuExpandableSubmenus: true, 
        relatedContainerForOverlayMenuSelector: '.top',
   };</script><script defer="defer" src="https://rdinnager.github.io/dinnage_lab_website/assets/js/scripts.min.js?v=f4c4d35432d0e17d212f2fae4e0f8247"></script><script>var images = document.querySelectorAll('img[loading]');

        for (var i = 0; i < images.length; i++) {
            if (images[i].complete) {
                images[i].classList.add('is-loaded');
            } else {
                images[i].addEventListener('load', function () {
                    this.classList.add('is-loaded');
                }, false);
            }
        }</script></body></html>